starknet_py/tests/e2e/docs/guide/test_deploying_new_contracts.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pytest
+from starknet_py.tests.e2e.utils import DevnetClientFactory
+
+directory = os.path.dirname(__file__)
+
+
+@pytest.mark.asyncio
+async def test_deploying_new_contracts(run_devnet):
+    # pylint: disable=import-outside-toplevel, disable=duplicate-code
+    # add to docs: start
```

Comment:
I've marked all code that should be in the docs:

# add to docs: start
code
code
# add to docs: end

code for test

# add to docs: start
...